boards/arm64/rcar_salvator_xs_m3/rcar_salvator_xs_m3.dts

```diff
@@ -30,3 +35,21 @@
 	pinctrl-names = "default";
 	status = "okay";
 };
+
+&emmc2 {
+	pinctrl-0 = <&emmc2_clk &emmc2_cmd &emmc2_ds
+		     &emmc2_data0 &emmc2_data1 &emmc2_data2 &emmc2_data3
+		     &emmc2_data4 &emmc2_data5 &emmc2_data6 &emmc2_data7>;
+	pinctrl-1 = <&emmc2_clk &emmc2_cmd &emmc2_ds
+		     &emmc2_data0 &emmc2_data1 &emmc2_data2 &emmc2_data3
+		     &emmc2_data4 &emmc2_data5 &emmc2_data6 &emmc2_data7>;
+	pinctrl-names = "default", "uhs";
+	disk {
```

Comment:
Why declare the disk here, if we are only going to enable it in overlays? Couldn't the disk be defined and enabled within the overlay itself?

Reply:
I would personally prefer we declare the disk in overlays, but I don't feel too strongly about it. Feel free to leave this as is